modules/logs/classes/class-log-db.php

```
@@ -301,16 +330,7 @@ private function erase_log_records( $start_time, $end_time ) {
-    public function is_site_action_log_existed( $site_id, $object_id ) {
-        global $wpdb;
-        return $wpdb->query( //phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery
-            $wpdb->prepare(
-                "SELECT `log_id`
-                FROM {$wpdb->mainwp_tbl_logs}
-                WHERE `site_id` = %d AND `object_id` = %s LIMIT 1 ",
-                $site_id,
-                $object_id
-            )
-        );
+    public function is_site_action_log_existed( $site_id = false, $object_id = false ) { //phpcs:ignore -- compatible.
```

PHPMD (2.15.0)

The method is_site_action_log_existed has a boolean flag argument $site_id, which is a certain sign of a Single Responsibility Principle violation. (Clean Code Rules)

(BooleanArgumentFlag)